__tests__/integration.test.js

```diff
@@ -104,4 +104,39 @@ describe('/api/articles', () => {
         }) 
     })
 })
-
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: 200, returns an array of comment objects asociated with the article ID', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const { comments } = body;
+            expect(comments).toHaveLength(11)
```

Comment:
Just a check for the length on it's own isn't enough to give confidence that we have all the data we need in the response,
you can make this test more robust by adding in a `forEach` loop to check the shape of each comment object